From d0edbd04e2df9412310a704495353cc59a8a450b Mon Sep 17 00:00:00 2001
From: Felix Kuehling <Felix.Kuehling@amd.com>
Date: Mon, 4 Jul 2016 11:42:26 -0400
Subject: [PATCH 1600/1722] drm/amdgpu: Enable file-backed userptr mappings for
 KFD

This change was originally applied in aaad19ff but lost in the
transition to kernel 4.4. Reapply it.

Change-Id: I2320429935c5311c2817af166e0553fcaf73d6ec
Signed-off-by: Felix Kuehling <Felix.Kuehling@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
index 85ddaf4..d84b74e 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_amdkfd_gpuvm.c
@@ -371,8 +371,7 @@ static int __alloc_memory_of_gpu(struct kgd_dev *kgd, uint64_t va,
 				alloc_domain, size);
 
 	if (userptr) {
-		ret = amdgpu_ttm_tt_set_userptr(bo->tbo.ttm, user_addr,
-						AMDGPU_GEM_USERPTR_ANONONLY);
+		ret = amdgpu_ttm_tt_set_userptr(bo->tbo.ttm, user_addr, 0);
 		if (ret) {
 			dev_err(adev->dev,
 				"(%d) failed to set userptr\n", ret);
-- 
2.7.4

